### PR TITLE
Consolidate perl modules and make our own glob wrapper

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Comment by dot-asm** [comment]
CIs are very angry:-)

**Comment by levitte** [comment]
I'll have a look some time tonight or on friday

**Comment by levitte** [comment]
Ok, small correction, let's see what the CIs say now...

**Comment by levitte** [comment]
There, CIs are happy now

**Comment by dot-asm** [comment]
I tried to twist it... I don't know if it's related, but test_ca fails if I have spaces in directories' names. Then CentOS used to have problem with run_tests.pl and I used to run custom-patched one (yeah, I should create merge request). After re-patching it fails at bunch of tests and then hangs in test_sslmessages. My Solaris VM used to be touchy, so I need to test that too... In other words I'll do some more tests...

**Comment by dot-asm** [comment]
BTW, why is it labelled master only? I'd say that it either should be 1.1.0 as well or :bsd_glob cherry-pick to 1.1.0 reverted...

**Comment by dot-asm** [comment]
Spaces in pathnames is unrelated and addressed in #4077.

**Comment by levitte** [comment]
Frankly, I'd rather revert the `:bsd_glob` change in 1.1.0.  This consolidation is much more of a change than I think I'm willing to put on 1.1.0-

**Comment by dot-asm** [comment]
+1 for revert.

**Comment by levitte** [comment]
0401110073cd392602855f9b72af2ebec7909625 reverted in 1.1.0: ef6d11f76fba7c9d4da821947f67acc5d603b99d

**Comment by levitte** [comment]
(I noticed the Travis builds didn't pass...  the error didn't seem to be related with this PR, but I rebased anyway, just for kicks)

**Comment by dot-asm** [comment]
H-m-m-m. As it turns out "my" CentOS problems go deeper than this, in sense that it's problematic even with current master. So I'm going to +1 this and proceed solve "my" problems by myself.

File: Configure [hunk 1]
```diff
@@ -12,10 +12,12 @@
 use 5.10.0;
 use strict;
 use Config;
+use FindBin;
+use lib "$FindBin::Bin/util/perl";
```
Comment (dot-asm) [inline Configure]: I suppose this flies on VMS... I mean slashes, and you don't have to build path with some platform-neutral call...

**Review by dot-asm — APPROVED** [review]

**Comment by levitte** [comment]
There was an error on VMS, now fixed.  Needs reconfirm

**Comment by levitte** [comment]
Runs correctly on VMS now

**Review by dot-asm — APPROVED** [review]

**Comment by dot-asm** [comment]
On side note, I'd appreciate if "ready" label was removed in cases when further feedback is requested :-)

**Comment by levitte** [comment]
Ah, right, forgot that.

**Comment by levitte** [comment]
Squashed and merged.  cb6afcd6ee0c0d66fae62e13fe5966171992f81c and 8d2214c0a49584044d96b80e846ac8f6df35a0ad

**Pull request closed without merging**